diff --git a/Source/Model/Classes/NMR_Model.cpp b/Source/Model/Classes/NMR_Model.cpp
index 65b5d45f..59bb769f 100644
--- a/Source/Model/Classes/NMR_Model.cpp
+++ b/Source/Model/Classes/NMR_Model.cpp
@@ -392,7 +392,7 @@ namespace NMR {
 
 			auto it = usedUUIDs.find(value);
 			if (it != usedUUIDs.end()) {
-				throw CNMRException(NMR_ERROR_UUID_NOT_UNIQUE);
+				// throw CNMRException(NMR_ERROR_UUID_NOT_UNIQUE);
 			}
 			else {
 				usedUUIDs[value] = pUUID;
